src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -158,6 +159,145 @@ def _add_imaging_plane(nwbfile: NWBFile, metadata=dict) -> NWBFile:
     return nwbfile
 
 
+def get_plane_segmentation_metadata(
+    metadata: dict,
+    plane_segmentation_index: int = 0,
+) -> dict:
+    """
+    Returns the metadata for the plane segmentation specified by the index.
+    The metadata is located in metadata["Ophys"]["PlaneSegmentation"][plane_segmentation_index].
+
+    Parameters
+    ----------
+    metadata : dict
+        The metadata in the nwb conversion tools format.
+    plane_segmentation_index : int, optional
+        The metadata in the nwb conversion tools format is a list of the different plane segmentations to add.
+        Specify which element of the list with this parameter, by default 0.
+
+    Returns
+    -------
+    dict
+        The metadata for this plane segmentation.
+    """
+    # Set the defaults and required infrastructure
+    metadata_copy = deepcopy(metadata)
+    default_metadata = get_default_ophys_metadata()
+    metadata_copy = dict_deep_update(default_metadata, metadata_copy, append_list=False)
+
+    image_segmentation_metadata = metadata_copy["Ophys"]["ImageSegmentation"]
+    plane_segmentation_metadata = image_segmentation_metadata["plane_segmentations"][plane_segmentation_index]
+
+    return plane_segmentation_metadata
+
+
+def _get_output_from_segmentation_extractor(
```

Comment:
I feel that output is not specific enough as a name for this function. That said, I also don't have a clear idea of how to name it. What is it doing is extracting the columns of the dynamic table that is to be created, so maybe that's a hint? maybe `extract_roi_table_data_from_segmentation_extractor` or something like that?

On the other hand, look at the comments below, I think that this function should be called from inside `add_plane_segmentation`. In that context, I am not sure that you require abstracting away this operations (i.e. making a function). I have been struggling with this type of decisions myself, check this out:

https://matthewrocklin.com/blog/work/2019/06/23/avoid-indirection